assets/static/js/tree.js

```diff
@@ -50,7 +50,13 @@ function insertTree(tree) {
         treeHTML += '            <select id="treetype-selector" onchange="' + on_filter_by_change + 'trees[\'' + tree.name + '\'].selected=null;trees[\'' + tree.name + '\'].type=this.value;updateTree(trees[\'' + tree.name + '\']);">';
         treeHTML += '                <option value="ra"';
         if (tree.type == "ra") treeHTML += ' selected';
-        treeHTML += '>Research Award</option>';
+        if (typeof PELAGOS_TEMPLATE_PROPS !== 'undefined'
```

Comment:
We can still use the utils/template-switch.js, we need to add it to the 'assets/js/app.js' and 'assets/js/nas-app.js'

`import templateSwitch from 'vue/utils/template-switch';
global.templateSwitch = templateSwitch;`


and then we can use the method `templateSwitch.getProperty(name)`